#src/scss/blocks/header.scss

```diff
@@ -1,3 +1,12 @@
-h1 {
-    color: orange;
-}
\ No newline at end of file
+.header__row {
+    margin-left: -51px;
+    margin-right: auto;
+}
+.header__logo {
+    margin-top: 30px;
+}
+.header__nav {
+    position: relative;
+    left: 41.66667%;
```

Comment:
weird magic numbers

Reply:
Your `.header__row` is given `display: flex` by bootstrap's `.row`
Elements with flex type of display have special positioning and aligning properties
use `justify-content: space-between;` on `.header-row` and remove this weird numbers, because these won't act the same on different size screens. while `justify-content: space-between;` always justifies childs with equal space between from left to right